src/Expander/Primitives.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS -Wno-name-shadowing #-}
```

Comment:
There was a warning about the `typeArgs` variable on line 129, which shadows some `typeArgs` global. I don't find name-shadowing warnings to be particularly useful, so I decided to hush disable them instead of fixing them.

Reply:
I wish there was a warning for shadowing names with compatible types only. In the absence of that, I think this is reasonable.